README.md

```diff
@@ -23,23 +23,29 @@ cargo install cargo-wipe
 cargo wipe --help
 ```
 
-## target
+## Usage
+
+To delete all folders called `folder-name` folders run
 
 ```
-cargo wipe target
+cargo wipe folder-name
 ```
 
-- Use the `-i <path>` argument to ignore certain paths.
-- Add the `-w` flag to wipe all Rust `target` folders found. USE WITH CAUTION!
+where `folder-name` is one of `rust`, `target`, `node`, `node_modules`. For example:

```

Comment:
The way this library does naming might be confusing for the users. `target` and `node_modules` are folder names. `rust` and `node` are the programming languages that create those folders. e.g. `node` is just an alias for `node_modules`.

Is there a way to word this in a better way? Or maybe we should just get rid of the aliases?

Reply:
Oh right, yeah that's very confusing! Even after skimming the code I thought `rust` and `node` were folder names.

Yeah I would definitely get rid of one or the other. I'd probably favour keeping the `rust`/`node` aliases and ditching `node_modules`/`target` because:

1. `target` is a bit of a weird thing to ask for.
2. You might want to add languages at some point for which cleaning is more complicated than a single directory name. Or they might not have a fixed directory name (e.g. CMake).

Reply:
Yep. I completely agree with all your points. I've raised https://github.com/mihai-dinculescu/cargo-wipe/issues/17